lib/parser.js

```diff
@@ -225,15 +237,19 @@ var parser = {
                 } else if (node.parent && node.parent.type === 'VariableDeclarator') {
                   functionName = node.parent.id.name;
                 }
-                var fnoutLine = '\nconsole.log("debuguy,"' +
+                var fnoutLine = 'var DEBOUT="debuguy,"' +
```

Comment:
Sorry that I couldn't find the definition of 'DEBOUT', what does it mean actually?